cypress/objects/t2Page.js

```diff
@@ -0,0 +1,66 @@
+class T2Page {
+    visit() {
+        cy.visit('/'); 
+    }
+
+    getUserDropDown() {
```

Comment:
This and some of other below methods are are in `loginPage.js` file. This file should only contain methods that work with elements of time logging page